src/simtools/simtel/simulator_array.py

```diff
@@ -67,12 +67,13 @@ def _make_run_command(self, run_number=None, input_file=None):
         str
             Command to run sim_telarray.
         """
+        config_file_path = self.corsika_config.array_model.config_file_path
```

Comment:
Why not `self.corsika_config.array_model.get_config_directory()`? Isn't the file exported to the model directory? That is what I would expect (and suggest to do).

Reply:
Fixed.